contrib/eraser/datasets/preprocess_removalbench.py

```diff
@@ -78,7 +78,7 @@ def main() -> None:
         shard_path = args.out_dir / f"{args.prefix}-{i:04d}.tar"
         print(f"🧩 Writing {shard_path} with {end - start} samples...")
 
-        with tarfile.open(shard_path, "w") as tar:
+        with tarfile.open(shard_path, "w", dereference=True) as tar:
```

Comment:
Why did it work initially?

Reply:
It wasn't, files were appearing in the tar, but #19 has not been properly tested (same thing for `img_extension`)